scripts/triggers-end-to-end-tests/triggers/index.js

```diff
@@ -43,6 +55,18 @@ const app = initializeApp(
 const auth = getAuth(app);
 connectAuthEmulator(auth, `http://${process.env.FIREBASE_AUTH_EMULATOR_HOST}`);
 
+function invokeHttpsFunction(name) {
+  const url = `http://localhost:${[FUNCTIONS_PORT, FIREBASE_PROJECT, FUNCTIONS_REGION, name].join(
+    "/"
+  )}`;
+  return fetch(url);
+}
+
+exports.triggerHttpsFunction = functions.https.onRequest(async (req, res) => {
+  await invokeHttpsFunction("httpsv2reaction");
+  res.json({ triggered: true });
+});
```

Comment:
Similar to comment above - instead of having a http function which invokes another http functoin, it looks like we can cut this middle man and just have test script invoke the http function directly.

Was there problem with that approach?

Reply:
No problem, but I wanted to invoke the test functions in a similar way for consistency (and I lazily didn't see the direct invoking test at the bottom).